charts/stable/valheim/values.yaml

```diff
@@ -9,10 +9,10 @@ secretEnv:
   SERVER_PASS: secret
 
 env:
-  STATUS_HTTP: true
-  STATUS_HTTP_PORT: "{{ .Values.service.status.ports.status.port }}"
-  SUPERVISOR_HTTP: true
   SUPERVISOR_HTTP_PORT: "{{ .Values.service.main.ports.main.port }}"
+  SUPERVISOR_HTTP: true
+  STATUS_HTTP_PORT: "{{ .Values.service.status.ports.status.port }}"
+  STATUS_HTTP: true
```

Comment:
I don't know why this reordering change is being made.  It seems unnecessary.  I specifically addressed it in 9bf159179b7744c105b9afbe01f8139cb4544bd3 even.